ClinGen/clincoded
# Issues #1796, #2035: GCI (Approve Classification) - Secondary Approvers/Contributors + SOP Version

Steps to test - 

1. Login to interface, curate new Gene-Disease record in GCI.
2. Navigate to Classification Matrix, scroll to bottom and save.
3. Submit the Provisional Classification to reach the Approval Classification where the new form for Secondary Approvers and Contributors lives.
4. Approval Classification form should have the following inputs and labels: Affiliation Approver, Final Approval Date, and SOP Version. Affiliation Approver is the only required field in this form. 
5. The Secondary Approvers/Contributors form should toggle by clicking the "Acknowledge Other Contributors" button (button text changes to "Close Form").
6. Curation Contributors should have full list of all current Affiliations, while Curation Approvers has full list of VCEPs/GCEPs. User should be able to multi-select from the drop downs.
7. Save the "Approved Classification", ensuring that all of the input values are passed to the Approval snapshot below. 
8. Navigate to the Approval Summary, all of the entered values for the form should be present in the Evidence Summary. Ensure that the value chosen for SOP Version is correct in the hyperlink. If you inspect the props for this component, either `GeneDiseaseEvidenceSummary` or `GeneDiseaseEvidenceSummaryHeader`, the new properties and their values are within the `provisional` object.

Side note: Publishing the Approved Classification cannot be done locally. It must be done within an instance, found here: https://1796-mm-secondaryapprovers.demo.clinicalgenome.org
**EDIT:** I have stopped the instance from running until this is being reviewed, so please log into EC2 to start it back up again :)

Secondary Approvers/Contributors form is part of issue #1796 
SOP Version is part of issue #2035
